package.json

```diff
@@ -36,6 +33,7 @@
     "pug": "^3.0.0",
     "qs": "^6.9.4",
     "sequelize": "^5.22.3",
+    "sequelize-cli": "^5.5.1",
```

Comment:
Moved this to a normal dependency so it can be run on the prebuilt post pruned code